onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -100,7 +108,7 @@ BackendManager::BackendManager(SessionContext& session_context,
     }
   }
 
-  if (ModelHasSymbolicInputDims(subgraph)) {
+  if (ModelHasSymbolicInputDims(subgraph) && session_context_.shape.empty()) {
```

Comment:
Is shapeempty checking for upper bound, lower bound ?

Reply:
I think this portion of code needs to be rewritten... to be device agnostic and to have one approach for dynamism